frontend/src/RegisterPage/RegisterPage.tsx

```diff
@@ -0,0 +1,47 @@
+import {ChangeEvent, useState} from "react";
+import axios from "axios";
+import {Link, useNavigate} from "react-router-dom";
+
+export default function RegisterPage() {
+
+    const[username, setUsername] = useState("");
+    const[password, setPassword] = useState("");
+
+    const nav = useNavigate();
+
+    function onChangeRegistrationName(event:ChangeEvent<HTMLInputElement>){
+        setUsername(event.target.value);
+    }
+
+    function onChangeRegistrationPassword(event:ChangeEvent<HTMLInputElement>){
+        setPassword(event.target.value);
+    }
+
+    function register(){
+        axios.post("/api/users", {username, password})
+            .then(() => nav("/"));
+    }
+
+
+    //"type" bei Button nicht nötig, weil standardmäßig submit-funkt.
```

Comment:
Richtig -> ABER: 
Später bitte Kommentare in einen extra Branch, damit "beim Kunden" auf dem Main nur das landet was dort auch hin muss :-)